mailbox/api/src/test/java/org/apache/james/mailbox/events/EventBusProvider.java

```diff
@@ -57,6 +57,10 @@ public void event(Event event) {
         int numberOfEventCalls() {
             return calls.get();
         }
+
+        void clear() {
```

Comment:
Is that method used?

Reply:
yes, we need it to clear when test in EventBus concurrent test